server/services/database/profile.js

```diff
@@ -22,7 +22,28 @@ const getAllProfiles = () => {
 	return pool.query("SELECT * FROM profile").then((result) => result.rows);
 };
 
+const getProfileById = (profileId) => {
+	return pool
+		.query("SELECT * FROM profile WHERE profile_public_id=$1", [profileId])
+		.then((result) => result.rows);
```

Comment:
because getProfileByPublicId should return one profile, it should return result.rows[0]